src/base/utils/version.h

```diff
@@ -50,7 +50,7 @@ namespace Utils
         typedef Version<T, N, Mandatory> ThisType;
 
         constexpr Version()
-            : m_components {}
+            : m_components()
```

Comment:
IMO, it's redundant at all. Why not just remove it?

Reply:
Is this due to our building with C++ 11?

Reply:
>Why not just remove it?

No, it is not the same, IIRC `m_components()` & `m_components{}` will init to zero, otherwise `m_components` will have indeterminate values.

Same below.

Reply:
Can you try `m_components {{}}` here?

Reply:
Since all the code here uses brace-enclosed initializers, I still insist that it be done in this way (unless it is completely impossible).